core/templates/dev/head/components/HintEditorDirective.js

```diff
@@ -22,15 +22,16 @@ oppia.directive('hintEditor', [
       restrict: 'E',
       scope: {
         hint: '=',
-        getIndex: '&index',
+        getIndexPlusOne: '&index',
```

Comment:
Hm, should probably be &indexPlusOne in that case, right?

(Good idea, btw.)

Reply:
done